src/mlpack/methods/reinforcement_learning/q_networks/dueling_dqn.hpp

```diff
@@ -46,27 +44,27 @@ using namespace mlpack::ann;
  * @tparam ValueNetworkType The type of network used for value network.
  */
 template <
-  typename OutputLayerType = EmptyLoss<>,
-  typename InitType = GaussianInitialization,
-  typename CompleteNetworkType = FFN<OutputLayerType, InitType>,
-  typename FeatureNetworkType = Sequential<>,
-  typename AdvantageNetworkType = Sequential<>,
-  typename ValueNetworkType = Sequential<>
+  typename OutputLayerType = mlpack::ann::EmptyLoss<>,
+  typename InitType = mlpack::ann::GaussianInitialization,
+  typename CompleteNetworkType = mlpack::ann::FFN<OutputLayerType, InitType>,
+  typename FeatureNetworkType = mlpack::ann::Sequential<>,
+  typename AdvantageNetworkType = mlpack::ann::Sequential<>,
+  typename ValueNetworkType = mlpack::ann::Sequential<>
```

Comment:
```suggestion
  typename OutputLayerType = ann::EmptyLoss<>,
  typename InitType = ann::GaussianInitialization,
  typename CompleteNetworkType = ann::FFN<OutputLayerType, InitType>,
  typename FeatureNetworkType = ann::Sequential<>,
  typename AdvantageNetworkType = ann::Sequential<>,
  typename ValueNetworkType = ann::Sequential<>
```
(Same as the last file.  Again, not 100% sure if it will work, but I *think* it should. :))